spec/features/user_sees_list_of_parks_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+feature "user sees all the parks" do
+  scenario "unauthenticated user visits index" do
+    first_park = FactoryGirl.create(:park)
+    second_park = FactoryGirl.create(:park)
+    third_park = FactoryGirl.create(:park)
+
+    visit "/"
+
+    expect(page).to have_content(first_park.name)
+    expect(page).to have_content(second_park.name)
+    expect(page).to have_content(third_park.name)
+  end
+
+  scenario "authenticated user visits index" do
+    user_1 = FactoryGirl.create(:user)
+    login_as(user_1)
+    first_park = FactoryGirl.create(:park)
+    second_park = FactoryGirl.create(:park)
+    third_park = FactoryGirl.create(:park)
```

Comment:
you can move these above and do a before (:each) do to create the parks-- seems like it does not matter if the authenticated user or the visitor sees it because they are not creating the park in this test.